api/custom_auth/views.py

```diff
@@ -74,6 +79,8 @@ def post(self, request: Request) -> Response:
                 ephemeral_token=serializer.validated_data["ephemeral_token"],
             )
             serializer.user = user
+            if settings.AUTH_JWT_COOKIE_ENABLED:
+                return authorise_response(user, Response())
```

Comment:
Right, I was typing out some questions, because I didn't fully understand the behaviour here (and in the similar conditional statement above), but I think I understand it more now, although I still have some questions. 

As I understand it, we're probably adding some headers to the response so that the client has the JWT? What I'm still not sure about then is why we're returning an empty `Response` object, instead of doing something like: 

```python
response = self._action(serializer)
if settings.AUTH_JWT_COOKIE_ENABLED:
    return authorise_response(user, response)
return response
```

Reply:
```
response = self._action(serializer)
```

creates the classic token and returns it. We don't need that for the cookie auth.

Reply:
I guess this means that the API changes depending on an environment variable, which is slightly odd? But maybe it's ok.

Reply:
I guess we have similar cases elsewhere so yeah, this is fine.